src/main/resources/structure/applications/appservice/usecases.test.java.mustache

```diff
@@ -0,0 +1,28 @@
+package {{package}}.config;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class UseCasesConfigTest {
+
+    @Test
+    public void testUseCasesConfig() {
+        AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(UseCasesConfig.class);
+        String[] beanNames = context.getBeanDefinitionNames();
+        
+        boolean useCaseBeanFound = false;
+        for (String beanName : beanNames) {
+            if (beanName.matches("^.+UseCase$")) {
+                useCaseBeanFound = true;
+                break;
+            }
+        }
+        
+        assertTrue(useCaseBeanFound, "No UseCase beans found");
```

Comment:
Hi @juparog this unit test is failing, please fix it

![image](https://github.com/user-attachments/assets/8850022d-9711-46ed-aa5a-bc04bfb082d4)